notifique/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,13 @@
       android:label="@string/label_application"
       android:supportsRtl="true"
       tools:ignore="GoogleAppIndexingWarning">
+    <service android:name=".NotificationListener"
+             android:label="@string/service_name"
```

Comment:
I have no idea. lol
It was in the documentation like that, so I assumed it would be a good idea. I was hoping to figure that out when I get it running on my phone.